src/content_script.js

```diff
@@ -1,11 +1,8 @@
-import { installDevtools } from "./install-devtools";
-import { setupMountAndUnmount } from "./inspected-window-helpers/force-mount-unmount.js";
-import { setupOverlayHelpers } from "./inspected-window-helpers/overlay-helpers.js";
 import browser from "webextension-polyfill";
```

Comment:
Do we need this polyfill? Are the apis for `runtime.sendMessage()` and `runtime.getURL()` different in firefox versus chrome? It looks like the spec webextension-polyfill was based on is a working group that is no longer active. See following links:

https://developer.chrome.com/docs/extensions/reference/api/runtime#method-sendMessage
https://github.com/mozilla/webextension-polyfill
https://www.w3.org/community/browserext/

Reply:
based on this https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime, seems that we should be safe without it... but I just left it as it was there already, haven't really tested in firefox

Reply:
let's remove the unneeded polyfill

Reply:
tried, but chrome will not let it pass... so some extra changes are needed, tried different stuff but seems easier just to have it there and later fix it